app/src/routes/Games/containers/GamesContainer.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { getGames } from "../../../store/games";
+
+const propTypes = {
+    gamesList: PropTypes.array
+};
+
+const defaultProps = {
+    gamesList: []
+};
+
+const mapDispatchToProps = dispatch => ({
+    getGames: getGames(dispatch)
+});
+
+const mapStateToProps = (state) => {
+    if (state.games !== null) {
+        return {
+            gamesList: state.games.gamesList
+        };
+    } else {
+        return {
+            gamesList: []
+        };
+    }
+};
+
+const enhance = connect(mapStateToProps, mapDispatchToProps);
+
+class GamesContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = null;
+
+        this.getGames = this.getGames.bind(this);
+    }
+
+    getGames() {
+/*        this.props.getGames(({ ok }) => {
+            console.log("getGames w GamesContainer");
+            if (!ok) {
+                console.log("Getting list of games failed");
+            }
+        });*/
+    }
+
+    render() {
+        return (
+            <div className="gamesList">
+              {this.props.gamesList.map(game =>
+                <section key={game.name}  className="gameSection section--center mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
+                      <header className="imageSection section__play-btn mdl-cell mdl-cell--3-col-desktop mdl-cell--2-col-tablet mdl-cell--4-col-phone mdl-color--blue-grey-50 mdl-color-text--white">
+                        <img className="tileImage" src={game.imageUrl}/>
+                      </header>
+                        <div className="mdl-card mdl-cell mdl-cell--9-col-desktop mdl-cell--6-col-tablet mdl-cell--4-col-phone mdl-shadow--2dp">
+                          <div className="mdl-card__supporting-text">
```

Comment:
instead of using `<div className="mdl-card__supporting-text">` you could import Card elements from `react-mdl/lib`